### DISCUSSION TITLE
Transcription failed again

### DISCUSSION THREAD

**Discussion by wuzimi** [deepgram/community#145] [General help] [answered]
Hi, could you pls help me on this error? It worked well previously. The information is here.
![503error](https://github.com/deepgram/community/assets/120776424/3046ddd5-9bae-4d05-9eb7-55a54f265b2c)
![erro info](https://github.com/deepgram/community/assets/120776424/c98e8c84-b77f-49e7-9200-7773918cf7c7)

**Comment by ALEKS0805** [comment]
I have the same error since yesterday

**Reply by ALEKS0805** [reply]
@wuzimi did you find the fix ?

**Comment by wuzimi** [comment]
No.

ALEKS0805 ***@***.***> 于 2023年5月10日周三 22:17写道：

**Comment by jjmaldonis** [comment]
Hey @wuzimi and @ALEKS0805, thanks for the reports. I tried to reproduce the issue but wasn't able to, so we need some more info to track down what's going on.

Can you please upload the audio you're using so I can see if it is related to the audio file itself? If you don't want to upload the audio file here, you can submit it via [this form](https://help.deepgram.com/hc/en-us/requests/new) - please mention this github issue in your message so I can find your audio file.

Also, can you provide the (python?) code you're using? It looks like you're using Deepgram's Python SDK, so I tried to replicate your issue using the code below. The code below ran successfully, so there must be a more nuanced problem that we need to track down.

```python
from deepgram import Deepgram  # pip install deepgram-sdk
import asyncio
import json
import os


# Initialize the Deepgram SDK.
DEEPGRAM_API_KEY = os.environ["DEEPGRAM_API_KEY"]  # Your Deepgram API Key
deepgram = Deepgram(DEEPGRAM_API_KEY)


async def main(url: str):
    r = await deepgram.transcription.prerecorded(
        source={"url": url},
        options={
            "model": "whisper-large",
            "punctuate": True,
            "diarize": True,
            "language": "en-us",
        },
    )

    print(json.dumps(r["results"], indent=2))


if __name__ == "__main__":
    url = "https://res.cloudinary.com/deepgram/video/upload/v1663090404/dg-audio/AI_Show_afpqeh.m4a"
    asyncio.run(main(url=url))
```

![image](https://github.com/deepgram/community/assets/2326898/9384f49a-316f-4ccd-8d63-23a0d48f7314)

**Comment by wuzimi** [comment, marked as answer]
This is solved. Thanks!

Jason Maldonis ***@***.***> 于 2023年5月18日周四 22:53写道：
